torchaudio/csrc/ffmpeg/buffer.h

```diff
@@ -0,0 +1,24 @@
+#pragma once
+#include <torch/torch.h>
+#include <torchaudio/csrc/ffmpeg/ffmpeg.h>
+#include <deque>
+
+namespace torchaudio {
+namespace ffmpeg {
+
+class Buffer {
+  std::deque<torch::Tensor> chunks;
+  AVMediaType media_type;
```

Comment:
Shall we add reference to the variables?
```suggestion
  std::deque<torch::Tensor>& chunks;
  AVMediaType& media_type;
```

Reply:
It's just an enum, so it's cheap and safer to copy.